bin/pycbc_inspiral

```diff
@@ -25,6 +25,7 @@ from pycbc.types import TimeSeries, FrequencySeries, zeros, float32, complex64
 import pycbc.fft.fftw, pycbc.version
 import pycbc.opt
 import pycbc.weave
+import pycbc.injcutter
```

Comment:
This isn't one option, it's several, deciding various points of behaviour here, something like:

--injection-skip-templates-chirp-time-threshold is just too long.